tests/unit/ssg-module/test_rule_yaml.py

```diff
@@ -1,6 +1,7 @@
 import pytest
 
 import os
+import tempfile
```

Comment:
@matejak did you mean to add this import?

Reply:
Not really, good catch!